docs/docs/dev_docs/contracts/deploying.md

```diff
@@ -1 +1,60 @@
-See sandbox section?
\ No newline at end of file
+# Deploying contracts
+
+Once you have [compiled](./compiling.md) your contracts you can proceed to deploying them using `aztec-cli`.
+
+## Prerequisites
+- aztec-cli installed (go to [CLI main section](./main.md) for installation instructions)
+- contract artifacts ready (go to [Compiling contracts section](../contracts/compiling.md) for instructions on how to compile contracts)
+- aztec-sandbox running (go to [Sandbox section](../sandbox/main.md) for instructions on how to install and run the sandbox)
+
+## Deploy
+
+To deploy the contracts we use the `deploy` command form `aztec-cli`:
+
+```bash
+aztec-cli deploy /path/to/contract/abi.json
+```
+
+### Arguments
+This command takes 1 mandatory positional argument which is the path to the contract ABI file in a JSON format (e.g. `build/contracts/PrivateToken.json`). Alternatively you can pass here a name of a contract ABI as exported by `@aztec/noir-contracts` (run `aztec-cli example-contracts` to get the list).
+
+The command also takes the following optional arguments:
+- `-args <constructorArgs...>` (default: `[]`): Arguments to pass to the contract constructor.
+- `--rpc-url <string>` (default: `http://localhost:8080`): URL of the Aztec node to connect to.
+- `--public-key <string>` (default: `undefined`): Optional encryption public key for this contract.
+Set this only if this contract is expected to receive private notes (in such a case the public key is used during the note encryption).
+> **NOTE**: Contracts which are expected to receive notes are called account contracts.
```

Comment:
```suggestion
```

Let's drop this as discussed via Slack. Contracts like the Escrow one in the e2e can receive encrypted notes but are not accounts.